src/xrpld/rpc/handlers/LedgerEntry.cpp

```diff
@@ -84,44 +108,55 @@ doLedgerEntry(RPC::JsonContext& context)
         else if (context.params.isMember(jss::deposit_preauth))
         {
             expectedType = ltDEPOSIT_PREAUTH;
+            auto const& dp = context.params[jss::deposit_preauth];
 
-            if (!context.params[jss::deposit_preauth].isObject())
+            if (!dp.isObject())
             {
-                if (!context.params[jss::deposit_preauth].isString() ||
-                    !uNodeIndex.parseHex(
-                        context.params[jss::deposit_preauth].asString()))
+                if (!dp.isString() || !uNodeIndex.parseHex(dp.asString()))
                 {
                     uNodeIndex = beast::zero;
                     jvResult[jss::error] = "malformedRequest";
                 }
             }
+            // clang-format off
```

Comment:
Why turn off clang-format here?

Reply:
every line here is independent condition, and clang-format mess them up